pkg/execution/executor/executor.go

```diff
@@ -2200,6 +2227,7 @@ func (e *executor) handleGeneratorInvokeFunction(ctx context.Context, gen state.
 		),
 	)
 	span.Send()
+	defer span.End()
```

Comment:
Any reason to do this? `span.Send()` calls `span.End()` anyway and we don't add any details to the span later on.

Reply:
addressed in #1360